docker/docker_official_images/3.8.0/jvm/Dockerfile

```diff
@@ -79,9 +75,8 @@ RUN set -eux ; \
     cp /opt/kafka/config/log4j.properties /etc/kafka/docker/log4j.properties; \
     cp /opt/kafka/config/tools-log4j.properties /etc/kafka/docker/tools-log4j.properties; \
     cp /opt/kafka/config/kraft/server.properties /etc/kafka/docker/server.properties; \
-    rm kafka.tgz kafka.tgz.asc KEYS; \
-    apk del wget gpg gpg-agent; \
-    apk cache clean;
+    rm kafka.tgz; \
+    apk del wget gpg gpg-agent; 
```

Comment:
As a non-blocking suggestion, is there some simple command that could be run here to verify the installation?  One we often use is `some-command --version`, but I'm not sure if anything in Kafka has something like that?